adapters/repos/db/vector/hnsw/maintainance.go

```diff
@@ -48,7 +48,7 @@ func (h *hnsw) growIndexToAccomodateNode(id uint64, logger logrus.FieldLogger) e
 	h.pools.visitedListsLock.Lock()
 	h.pools.visitedLists.Destroy()
 	h.pools.visitedLists = nil
-	h.pools.visitedLists = visited.NewPool(1, len(newIndex)+500)
+	h.pools.visitedLists = visited.NewPool(1, len(newIndex)+512)
```

Comment:
I would put that `512` value into a `const` block that is here in this file, as we already keep all of the settings there

Reply:
This is on purpose in order to be consistent with the current implementation.
Also to me it's more readable and in addition the value is just a roughly estimation of number of lists.

Nevertheless this could be done as part of another refactoring ticket

Reply:
ok